src/fsharp/tast.fs

```diff
@@ -4561,7 +4552,7 @@ and
     [<DebuggerBrowsable(DebuggerBrowsableState.Never)>]
     member x.DebugText = x.ToString()
 
-    override x.ToString() = sprintf "TBind(%s, ...)" x.Var.CompiledName
+    override x.ToString() =  sprintf "TBind(%s, ...)" (x.Var.CompiledName None)
```

Comment:
@ dsyme … This is why CompilerGlobalSources is an option.
There is no way to get tcglobals to here,

Reply:
Nit: whitespace before `sprintf`.